### PR TITLE
Update black

### PR DESCRIPTION
**Pull request by fritzo**
1. `pip install --update black && make format`

### CODE REVIEW

**Comment by fritzo** [comment]
@fehiepsi it looks like the jax tests may be failing due to numpyro renaming of distributions? could you ptal?

**Comment by fehiepsi** [comment]
It looks like the issue is related to the recently added distribution RelaxedBernoulli. Let me tal.

File: test/test_distribution_generic.py [hunk 1]
```diff
@@ -297,6 +297,13 @@ def __hash__(self):
         funsor.Real,
     )
 
+    # Logistic
+    DistTestCase(
+        "dist.Logistic(loc=case.loc, scale=case.scale)",
+        (("loc", f"randn({batch_shape})"), ("scale", f"rand({batch_shape})")),
+        funsor.Real,
+    )
```
Comment (fehiepsi) [inline test/test_distribution_generic.py:305]: Just added this one while debugging RelaxedBernoulli for jax backend.

File: test/test_distribution_generic.py [hunk 2]
```diff
@@ -381,6 +388,7 @@ def __hash__(self):
         "dist.RelaxedBernoulli(temperature=case.temperature, logits=case.logits)",
         (("temperature", f"rand({batch_shape})"), ("logits", f"rand({batch_shape})")),
         funsor.Real,
+        xfail_reason="backend not supported" if get_backend() != "torch" else "",
```
Comment (fehiepsi) [inline test/test_distribution_generic.py:391]: RelaxedBernoulli is a transformed distribution but currently, only "torch" backend supports TransformedDistribution.

**Comment by fritzo** [comment]
Thanks for fixing the jax issues @fehiepsi!  Would you be able to approve this PR?

**Review by fehiepsi — APPROVED** [review]
LGTM.

**Pull request merged**